homeassistant/components/sensor/__init__.py

```diff
@@ -89,6 +89,12 @@
 
 SCAN_INTERVAL: Final = timedelta(seconds=30)
 
+_NON_NUMERIC_DEVICE_CLASSES = {
+    SensorDeviceClass.DATE,
+    SensorDeviceClass.ENUM,
+    SensorDeviceClass.TIMESTAMP,
+}
```

Comment:
Almost all of these things have been moved into `const.py`, I suggest moving this there too.

Reply:
Rename to `NON_NUMERIC_DEVICE_CLASSES` ?

Reply:
We have a bunch of tests that use these three too, which could use the constant now :)